lib/matplotlib/tri/triplot.py

```diff
@@ -42,11 +44,11 @@ def triplot(ax, *args, **kwargs):
     linestyle, marker, color = matplotlib.axes._base._process_plot_format(fmt)
 
     # Insert plot format string into a copy of kwargs (kwargs values prevail).
-    kw = kwargs.copy()
+    kw = cbook.normalize_kwargs(kwargs, mlines.Line2D)
     for key, val in zip(('linestyle', 'marker', 'color'),
                         (linestyle, marker, color)):
         if val is not None:
-            kw[key] = kwargs.get(key, val)
+            kw[key] = kw.get(key, val)
```

Comment:
What this is really saying is:
```suggestion
            kw.setdefault(key, val)
```

Reply:
Thanks. Made the changes.